src/react-components/ui-root.js

```diff
@@ -295,7 +295,7 @@ class UIRoot extends Component {
 
   onConcurrentLoad = () => {
     if (qsTruthy("allow_multi") || this.props.store.state.preferences["allowMultipleHubsInstances"]) return;
-    this.startAutoExitTimer("autoexit.concurrent_subtitle");
+    this.startAutoExitTimer("concurrent-session");
```

Comment:
Looks like this should be `"concurrentSession"`

https://github.com/mozilla/hubs/blob/redesign/exit-modals/src/react-components/room/AutoExitWarningModal.js#L9

Any thoughts on how we might be able to catch these (especially if they're infrequently used) at build time, given that the message id's will not change?

Reply:
Types :sweat_smile: But I think a string enum object will do the trick here.